lua/ttt2/libraries/entity_outputs.lua

```diff
@@ -52,7 +52,7 @@ function entityOutputs.RegisterHook(hookName)
     entityOutputs.hooks[hookName] = true
 
     hook.Add(hookName .. "_Internal", hookName .. "_name", function()
-        local activator, caller = ACTIVATOR, CALLER
+        local activator, caller = ACTIVATOR, CALLER -- FIXME: where are these defined? we should at least link to where these come from
```

Comment:
yes, they are. These are global variables, I forgot where I got that info from. Probably the gmod wiki or the valve dev wiki. It works 100% and it is a wild concept

Reply:
done in [7ed4347](https://github.com/TTT-2/TTT2/pull/1574/commits/7ed4347fafa536cdf05d130afe3b4dabd48c199e)